data/mystery_event_msg.s

```diff
@@ -13,332 +15,11 @@
-	waitbuttonpress
-	release
-	end
-
-MysticTicket_Obtained: @ 8675578
-	vmessage sText_MysticTicketGot
-	waitmessage
-	waitbuttonpress
-	release
-	end
-
-sText_MysticTicket2:
-	.string "Thank you for using the MYSTERY\nGIFT System.\pYou must be {PLAYER}.\nThere is a ticket here for you.$"
-
-sText_MysticTicket1:
-	.string "It appears to be for use at the\nLILYCOVE CITY port.\pWhy not give it a try and see what\nit is about?$"
-
-sText_MysticTicketGot:
-	.string "Thank you for using the MYSTERY\nGIFT System.$"
-
-sText_MysticTicketNoPlace:
-	.string "Oh, I'm sorry, {PLAYER}.\nYour BAG's KEY ITEMS POCKET is full.\pPlease store something on your PC,\nthen come back for this.$"
-
-MysteryEventScript_AlteringCave:: @ 86756E3
-	setvaddress MysteryEventScript_AlteringCave
-	addvar VAR_ALTERING_CAVE_WILD_SET, 1
-	compare_var_to_value VAR_ALTERING_CAVE_WILD_SET, 10
-	vgoto_if 0, MysteryEventScript_AlteringCave_
-	setvar VAR_ALTERING_CAVE_WILD_SET, 0
-MysteryEventScript_AlteringCave_: @ 86756FD
-	lock
-	faceplayer
-	vmessage sText_MysteryGiftAlteringCave
-	waitmessage
-	waitbuttonpress
-	release
-	end
-
-sText_MysteryGiftAlteringCave::
-	.string "Thank you for using the MYSTERY\nGIFT System.\pThere appears to be a rumor about\nrare POKéMON sightings.\pThe sightings reportedly came from\nthe ALTERING CAVE on ROUTE 103.\pPerhaps it would be worthwhile for\nyou to investigate this rumor.$"
-
-MysteryEventScript_OldSeaMap:: @ 86757F4
-	setvaddress MysteryEventScript_OldSeaMap
-	lock
-	faceplayer
-	checkflag FLAG_RECEIVED_OLD_SEA_MAP
-	vgoto_if 1, OldSeaMap_Obtained
-	checkflag FLAG_CAUGHT_MEW
-	vgoto_if 1, OldSeaMap_Obtained
-	checkitem ITEM_OLD_SEA_MAP, 1
-	compare_var_to_value VAR_RESULT, 1
-	vgoto_if 1, OldSeaMap_Obtained
-	vmessage sText_MysteryGiftOldSeaMap
-	waitmessage
-	waitbuttonpress
-	checkitemspace ITEM_OLD_SEA_MAP, 1
-	compare_var_to_value VAR_RESULT, 0
-	vgoto_if 1, OldSeaMap_NoBagSpace
-	giveitem_std ITEM_OLD_SEA_MAP
-	setflag FLAG_ENABLE_SHIP_FARAWAY_ISLAND
-	setflag FLAG_RECEIVED_OLD_SEA_MAP
-	vmessage sText_MysteryGiftOldSeaMap_1
-	waitmessage
-	waitbuttonpress
-	release
-	end
-
-OldSeaMap_NoBagSpace: @ 867584F
-	vmessage sText_MysteryGiftOldSeaMap_3
-	waitmessage
-	waitbuttonpress
-	release
-	end
-
-OldSeaMap_Obtained: @ 8675858
-	vmessage sText_MysteryGiftOldSeaMap_2
-	waitmessage
-	waitbuttonpress
-	release
-	end
-
-sText_MysteryGiftOldSeaMap:
-	.string "Thank you for using the MYSTERY\nGIFT System.\pLet me confirm--you are {PLAYER}?\pWe received this OLD SEA MAP\naddressed to you.$"
-
-sText_MysteryGiftOldSeaMap_1:
-	.string "It appears to be for use at the\nLILYCOVE CITY port.\pWhy not give it a try and see what\nit is about?$"
-
-sText_MysteryGiftOldSeaMap_2:
-	.string "Thank you for using the MYSTERY\nGIFT System.$"
-
-sText_MysteryGiftOldSeaMap_3:
-	.string "Oh, I'm sorry, {PLAYER}.\nYour BAG's KEY ITEMS POCKET is full.\pPlease store something on your PC,\nthen come back for this.$"
+	.include "data/scripts/meventmsg_stamp_card.inc"
```

Comment:
Why is `msg` in the name of all of these?  These are full mystery event scripts, so I don't think `msg` is appropriate in the names.

Reply:
Only for naming consistency with `mystery_event_msg.s`, which only contains those split files. I can rename that as well